managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedCursorImpl.java

```diff
@@ -2678,8 +2676,7 @@ private void persistPositionMetaStore(long cursorsLedgerId, PositionImpl positio
         ManagedCursorInfo.Builder info = ManagedCursorInfo.newBuilder() //
                 .setCursorsLedgerId(cursorsLedgerId) //
                 .setMarkDeleteLedgerId(position.getLedgerId()) //
-                .setMarkDeleteEntryId(position.getEntryId()) //
-                .setLastActive(lastActive); //
+                .setMarkDeleteEntryId(position.getEntryId()); //
```

Comment:
Please keep `setLastActive` for compatibility.

Reply:
Done, thanks for review.